src/conventional/metar_csv2ioda.py

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
```

Comment:
@mmiesch, @srherbener Do we need ```env python3``` or ```env python```?

Reply:
We want to explicitly declare python3 since we are moving to drop support for python2 (which was deprecated last January). We already have this captured in issue #294.

Reply:
Fixed, so changing item to resolved.